docs/user/YAL5-codon-pairs_usage.md

```diff
@@ -0,0 +1,53 @@
+# Running YAL5-codon-pairs.R
+
+## Input ##
+The script `YAL5-codon-pairs.R` looks into metafeatures, currently the translational landscape surrounding codon pairs. 
+
+- h5 file
+- gff3 file
+- tsv codon positions file
+
+Read counts are fetched from the H5 file of a sample and assigns the counts to the cooresponding codon pair position for the genes using the species-specific GFF3 file and a tsv file containing the codon positions for all genes of the species being studied, i.e. the `yeast_codon_table.tsv`. The script calculates the normalised number of reads within a window surrounding each occurrence of the feature of interest (e.g. a codon pair) which are then overlayed to generate a single plot. Normalisation is carried out within a set window of codons adjacent to the feature of interest. The default value of this window is currently 5 codons either side of the feature of interest. 
+The aim of this script is to study translation by examining whether specific features can slow down translation, thereby having an inhibitory effect. If the feature is being translated more slowly than its surrounding codons this will generate a peak. 
+
+## Outputs ##
+There is currently one output when running YAL5-codon-pairs.R and one that is being tested.
+
+The first is a pdf of a graph plotting the average relative number of reads mapping to each relative position within the desired window of the feature of interest. This is produced when one feature is given, for example the codon pair 'CGA GCC'.
```

Comment:
`each relative position` -> `each position relative to the feature of interest` ?